planet/cli/subscriptions.py

```diff
@@ -29,6 +30,17 @@ def check_item_types(ctx, param, item_types) -> Optional[List[dict]]:
         raise click.BadParameter(str(e))
 
 
+def check_geom(ctx, param, geometry) -> Optional[dict]:
```

Comment:
This method looks like it's defined in both `cli/data.py` and `cli/subscriptions.py`.  Is there a place where we can define `check_geom()` once and share it as needed across different files?

Reply:
that seemed to be the pattern for other cli methods. [check_item_type/s](https://github.com/planetlabs/planet-client-python/blob/main/planet/cli/data.py#L73) in data, [check_item_type/s](https://github.com/planetlabs/planet-client-python/blob/main/planet/cli/subscriptions.py#L21) in subs, both loops through and call a validator in another file.

Reply:
Hmm, that's a pattern I think we should move away from where possible.  Would you please move `check_geom()` to a place where it could be shared?  I believe that following DRY patterns will help make the SDK more maintainable going forward.

Reply:
moved! I started moving check_item_type and check_item_types as well, but that turned into a can of 🪱s

Reply:
Thanks!  No worries about check_item_type/s, I would consider those methods to be outside the scope of this PR.